File: src/main/java/org/shark/example/dao/repository/pojo/JsonJoinExampleDo.java
package org.shark.example.dao.repository.pojo;


import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "JSON_JOIN_EXAMPLE")
@Data
public class JsonJoinExampleDo {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    @Column(name = "ID")
    private Integer id;

    @Column(name = "JSON_EXAMPLE_ID")
    private Integer jsonExampleId;

    @Column(name = "DESCRIPTION")
    private String description;
}

File: src/main/kotlin/org/shark/example/dao/repository/pojo/JsonJoinExampleDo.kt
package org.shark.example.dao.repository.pojo

import jakarta.persistence.*

@Entity
@Table(name = "JSON_JOIN_EXAMPLE")
data class JsonJoinExampleDo(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID")
    var id: Int? = null,

    @Column(name = "JSON_EXAMPLE_ID")
    var jsonExampleId: Int? = null,

    @Column(name = "DESCRIPTION")
    var description: String? = null
)
